pennylane/measurements.py

```diff
@@ -596,9 +597,7 @@ def circuit(x):
         QuantumFunctionError: `op` is not an instance of :class:`~.Observable`
     """
     if not op.is_hermitian:
-        raise qml.QuantumFunctionError(
-            f"{op.name} is not an observable: cannot be used with expval"
-        )
+        warnings.warn(f"{op.name} might not be an observable.")
```

Comment:
```suggestion
        warnings.warn(f"{op.name} might not be an Hermitian.")
```

Since "Observable" is a particular type of base class that we are moving away from relying on, let's use "Hermitian" here instead. Same below.

Reply:
Done!